BLSQ/iaso
# POLIO-1023: confirm button not disabled on save

Explain what problem this PR is resolving
- When the form is submitted on campaign creation or update , the save button is not disabled
Related JIRA tickets : [POLIO-1023](https://bluesquare.atlassian.net/browse/POLIO-1023)

## Self proofreading checklist


## Changes
- Put values and helpers in the formik onSubmit method as parameters
- Set submitting state to true when the save button in clicked
- Set submitting state to false when an error occurred

## How to test
- Go to campaign
- Edit or create a campaign
- Check if the save button is immediatly disabled

## Print screen / video
![Peek 2023-06-06 16-44](https://github.com/BLSQ/iaso/assets/19631540/f8a2af02-3562-43e7-8da7-e553a22d302b)



[POLIO-1023]: https://bluesquare.atlassian.net/browse/POLIO-1023?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
